Exercise_6/ReversePolishNotation.js

```diff
@@ -0,0 +1,44 @@
+// const process = require('process');
+// console.log(calculate(process.argv[2]));
+
+function calculate(argument,extraOperators) {
```

Comment:
Please use `reformat code`. You forgot to add a space here: `argument, extraOperators`.